fluent-plugin-carbon-v2/README.md

```diff
@@ -22,22 +55,49 @@ gem "fluent-plugin-carbon-v2"
 
 And then execute:
 
-```
-$ bundle
+```sh
+bundle
 ```
 
 ## Configuration
 
-You can generate configuration template:
+### relabel (hash) (optional)
 
-```
-$ fluent-plugin-config-format filter carbon-v2
-```
+Relabel the field name in the input record.
+For every (`key`, `value`) pair in the hash, the field named with `key` in the input record will be relabeled to `value`.
+If `key` is not a field in the input record, it will be ignored.
+If `value` is an empty string, the field will be removed from the record.
+
+Default value: `{}`.
+
+### inclusions (hash) (optional)
+
+Whitelist of the records with regular expression matching on the field(s).
+For __all__ (`key`, `value`) pairs in the hash, only following records will be included in the output:
+
+- the value of field named with `key` matches the `value` (as regular expression).
+
+Default value: `{}`.
+
+### strict_inclusions (bool) (optional)
+
+If `true`, records missing any field in keys of `inclusions` will be dropped.
+
+Default value: `false`
+
+### exclusions (hash) (optional)
+
+Blacklist of the records with regular expression matching on the field(s).
+For __any__ (`key`, `value`) pair in the hash, following records will be excluded in the output:
+
+- the value of field named with `key` matches the `value` (as regular expression).
+
+Default value: `{}`.
+
+### strict_exclusions (bool) (optional)
 
-You can copy and paste generated documents here.
+If `true`, records missing any field in keys of `exclusions` will be dropped.
```

Comment:
Is this the behaviour we want for `strict_exclusions`? In my mind, `strict_exclusions` would drop any records that include any field in keys of `exclusions`, rather than dropping the records that are missing those fields.

Reply:
I may not very follow your statement here - drop any records that include any field in the keys of exclusions. 

Let's take a look case `exclusions = {"a" => "va", "b" => "vb"}` and `strict_exclusions = true`, so you think any record contains "a" or "b" will be dropped? so the value of `exclusions` mapping is useless here?

In my thought, on above case it should means record must have both "a" and "b" fields and `record["a"] != "va" && record["b"] != "vb"`

Reply:
I see what you mean, thanks for clarifying. I agree that this is probably the more useful use case.